src/ballot-forms.php

```diff
@@ -65,7 +65,30 @@
 </head>
 
 <body>
-  
+
+<?php 
+      $personality = '';
+      if ($org_acronym == 'acap'){
+          $personality = "Psychmates";
+      } else if ($org_acronym == 'aeces'){
+          $personality = "AECES";
+      } else if ($org_acronym == 'elite'){
+          $personality = "Enthusiasts";       
+      } else if ($org_acronym == 'give'){
+          $personality = "Educators";      
+      } else if ($org_acronym == 'jehra'){
+          $personality = "JEHRA";    
+      } else if ($org_acronym == 'jpia'){
+          $personality = "JPIANs";   
+      } else if ($org_acronym == 'piie'){
+          $personality = "IEhinyero";  
+      } else if ($org_acronym == 'jmap'){
+          $personality = "Marketista"; 
+      } else if ($org_acronym == 'sco'){
+          $personality = "Iskolar"; 
+      }
+?>
+
```

Comment:
You can move this to the organization-list.php file. Let's strive for a cleaner and modular code.